js/convert.js

```diff
@@ -99,7 +99,7 @@ const YNABAccountData = function (accountNumber) {
         }
 
         const date = new Date().toJSON().slice(0,10).replace(/-/g,"\/");
-        const fileName = accountNumber + "_" + date + ".csv";
+        const fileName = "ynab__" + accountNumber + "_" + date + ".csv";
```

Comment:
```suggestion
        const fileName = "ynab_" + accountNumber + "_" + date + ".csv";
```

A single underscore would be better.